src/react.tsx

```diff
@@ -14,24 +39,61 @@ export function connect<StoreState extends object>(store: StoreDefinition<StoreS
     Component: React.ComponentType<PropsWithStore>
   ): React.ComponentClass<Diff<PropsWithStore, { store: Store<StoreState> }>> {
 
+    type SubscriptionKeys = keyof StoreState | '<all>'
```

Comment:
I don't think anybody is going to create a field named `<all>`, probably JS won't allow it. it will break object definitions as you can't do
```
  type State = {
    <all>: string
  };
```